### ISSUE TITLE
Unable to re-enable Bink videos on startup without modding and rebuilding Chimera

### ISSUE THREAD

**Issue by deuxsonic** [SnowyMouse/chimera#122]
So I've been having a Chimera issue for years and many different versions where even with startup videos enabled in chimera.ini (intro_videos=1), they still will not start, so it doesn't seem to care what the setting is, it will use enable_novideo regardless. I realize it is an annoyance fix for many, but if the option is given to enable or disable the behavior, the option should work. I've encountered the issue on different computers with different configurations. I have sidestepped the issue by recently rebuilding Chimera on my end by commenting out "enable_novideo();" (line 282) in chimera.cpp to stop this functionality from being used, but this doesn't actually fix the incorrect behavior of Chimera per se.

**Comment by MangoFizz** [comment]
What version of the game are you trying this on? Are the .bik files in the game folder?

**Comment by deuxsonic** [comment]
Custom Edition 1.10. Yes, the Bink files are in the game's root.

**Comment by Aerocatia** [comment]
The setting is working as intended for me. Are you sure it is uncommented properly in the config?

**Comment by deuxsonic** [comment]
It is uncommented in chimera.ini. I figured it worked fine for others, especially since most people would operate with it skipping videos, so most people wouldn't notice if it was always doing this erroneously. I thought maybe it could be some kind of race condition, like the ini settings are read in after the decision to use enable_novideo is already set, but only sometimes?

**Comment by PseudoProxen** [comment]
i hate to ask, you're not using a shortcut with -novideo in it by chance

**Comment by MangoFizz** [comment]
> Custom Edition 1.10. Yes, the Bink files are in the game's root.

Have you tested if it works in the retail version? Can you attach a screenshot of the game's folder?

**Comment by deuxsonic** [comment]
I'm not using a shortcut to launch the game, I launch haloce.exe directly. I haven't tested this with the retail version of the game, but I could do so. I've included a screenshot of the game's root. I didn't mention it before but my system is Windows 10 Enterprise x64 19045.5073.
![haloroot](https://github.com/user-attachments/assets/8236e5ec-c6e2-45e8-8a6b-6d6414fc5de4)

**Comment by PseudoProxen** [comment]
Wondering if Rad Tools ( icon on the bink videos) or some codec is interfering.

**Comment by deuxsonic** [comment]
If that were the case, wouldn't it also interfere with my source code change that bypasses the novideo stuff?

**Comment by PseudoProxen** [comment]
Maybe I should ask first, do the videos play on normal vanilla startup?

Just to weed out any suspicion of the codec issue.

Also are you using any additional mods/addons?

**Comment by deuxsonic** [comment]
The videos do play on vanilla CE. The only mod that I use is Refined.

**Comment by PseudoProxen** [comment]
yeah this one is a weird one, and you're using the latest Chimera and HaloCE Executable?

Heard of binkw32.dll causing issues due to people screwing with bik files or utilizing different binkw32 Dynamic Libraries. But Chimera being the cause I am stumped.

**Comment by deuxsonic** [comment]
Yep, original binkw32.dll and video files and latest Chimera and 1.10 CE. That's why I thought some kind of race condition where the setting isn't read until after the point where the setting would have effect, as simply disabling the skip in the source code fixes the problem or some other interaction.
